pkg/lib/dynatrace/dynatrace.go

```diff
@@ -861,6 +930,14 @@ func (ph *Handler) QueryDynatraceDashboardForSLIs(keptnEvent *common.BaseKeptnEv
 			continue
 		}
 
+		if tile.TileType == "SLO" {
+			// we will take the SLO definition from Dynatrace
+			for _, sloEntity := range tile.AssignedEntities {
+				ph.Logger.Debug(fmt.Sprintf("Processing SLO Definition: %s", sloEntity))
```

Comment:
can you please explain this section? currently there is only a log statement for each entity, but no actual processing is done here

Reply:
This is going to be for a future implementation of supporting Dynatrace SLOs. I added this to the code so that we don't forget to implement it. The actual implementation however will be done in probably a week or two once this feature becomes available. If you think we should remove this and add it on a different branch that is just focused on Dynatrace SLO support I am also fine with taking it out

Reply:
i removed that code with my latest push